src/main/java/space/obminyashka/items_exchange/service/UserService.java

```diff
@@ -109,6 +108,15 @@ public interface UserService {
      */
     boolean isPasswordMatches(User user, String encodedPassword);
 
+    /**
+     * Given a username and an encoded password, return true if the encoded password matches the user's password, false
+     * otherwise.
+     * @param username The username of the user.
+     * @param encodedPassword The encoded password that you want to check against the user's password.
+     * @return A boolean value.
+     */
+    boolean isUserPasswordMatches(String username, String encodedPassword);
```

Comment:
Incorrect parameter name: "encodedPassword" it's a new user's password which is definitely not encoded at all

Reply:
rename